library/Ivoz/Provider/Domain/Model/CarrierServer/CarrierServer.php

```diff
@@ -108,4 +108,13 @@ public function getName()
                 $this->getId()
             );
     }
+
+    protected function setAuthPassword($authPassword = null)
+    {
+        if ($authPassword) {
+            $authPassword = trim($authPassword);
+        }
```

Comment:
This check is not in the other setters that allow nullable arguments (like Terminals, RetailAccounts and so ?